### PR TITLE
update glossary entry for MVP 

### PR DESCRIPTION
**Pull request by Richabanker**
This PR adds a hyperlink to `feature gates` in the glossary entry for MVP

### CODE REVIEW

File: content/en/docs/concepts/architecture/mixed-version-proxy.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ from the upgrade process.
 This mechanism is called the _Mixed Version Proxy_. 
 
 ## Enabling the Mixed Version Proxy
-Ensure that `UnknownVersionInteroperabilityProxy` [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) 
+Ensure that {{< glossary_tooltip text="MVP" term_id="mixed-version-proxy" >}} [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/)
```
Comment (dipesh-rawat) [inline content/en/docs/concepts/architecture/mixed-version-proxy.md:26]: We prefer to use site-relative links where feasible.

```suggestion
Ensure that {{< glossary_tooltip text="MVP" term_id="mixed-version-proxy" >}} [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) 
```

File: content/en/docs/reference/glossary/mixed-version-proxy.md [hunk 2]
```diff
@@ -17,5 +17,5 @@ When a cluster has multiple API servers running different versions of Kubernetes
 feature enables resource requests to be served by the correct API server.
 
 MVP is disabled by default and can be activated by enabling
-the feature gate named `UnknownVersionInteroperabilityProxy` when 
-{{< glossary_tooltip text="API Server" term_id="kube-apiserver" >}} is started.
+the [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) named `UnknownVersionInteroperabilityProxy` when
```
Comment (dipesh-rawat) [inline content/en/docs/reference/glossary/mixed-version-proxy.md:20]:
```suggestion
the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) named `UnknownVersionInteroperabilityProxy` when 
```

File: content/en/docs/concepts/architecture/mixed-version-proxy.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ from the upgrade process.
 This mechanism is called the _Mixed Version Proxy_. 
 
 ## Enabling the Mixed Version Proxy
-Ensure that `UnknownVersionInteroperabilityProxy` [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) 
+Ensure that {{< glossary_tooltip text="MVP" term_id="mixed-version-proxy" >}} [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
```
Comment (sftim) [inline content/en/docs/concepts/architecture/mixed-version-proxy.md:26]: The original text is better; there's no feature gate named `MVP`. Also, you don't need a tooltip in the page about the concept.

File: content/en/docs/concepts/architecture/mixed-version-proxy.md [hunk 4]
```diff
@@ -58,9 +58,9 @@ these fields in the API server configuration file.
 If these flags are unspecified, peers will use the value from either `--advertise-address` or
 `--bind-address` command line argument to the kube-apiserver. If those too, are unset, the host's default interface is used.
 
-## Mixed version proxying
+## Mixed Version Proxying
```
Comment (sftim) [inline content/en/docs/concepts/architecture/mixed-version-proxy.md:61]: All headings in that file should be in title case, and Mixed Version Proxy should be mixed version proxy (italicized on first use).

File: content/en/docs/concepts/architecture/mixed-version-proxy.md [hunk 5]
```diff
@@ -58,9 +58,9 @@ these fields in the API server configuration file.
...
 
-When you enable mixed version proxying, the [aggregation layer](/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/) 
+When you enable {{< glossary_tooltip text="MVP" term_id="mixed-version-proxy" >}}, the [aggregation layer](/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/)
```
Comment (sftim) [inline content/en/docs/concepts/architecture/mixed-version-proxy.md:63]: You don't need a tooltip in the page about the concept.

File: content/en/docs/reference/glossary/mixed-version-proxy.md
Comment (sftim) [inline content/en/docs/reference/glossary/mixed-version-proxy.md:1]: BTW the `full_link` is wrong (omit the incorrect fragment identifier; add a trailing `/`)

File: content/en/docs/concepts/architecture/mixed-version-proxy.md (hunk 3 above)
Comment (Richabanker) [reply content/en/docs/concepts/architecture/mixed-version-proxy.md]: Reverted.

File: content/en/docs/reference/glossary/mixed-version-proxy.md
Comment (Richabanker) [reply content/en/docs/reference/glossary/mixed-version-proxy.md:1]: done

File: content/en/docs/concepts/architecture/mixed-version-proxy.md (hunk 4 above)
Comment (Richabanker) [reply content/en/docs/concepts/architecture/mixed-version-proxy.md]: Done

File: content/en/docs/concepts/architecture/mixed-version-proxy.md (hunk 5 above)
Comment (Richabanker) [reply content/en/docs/concepts/architecture/mixed-version-proxy.md]: Removed, I misunderstood from earlier that we wanted to use the tooltip in the concept page. Hence made all the references in here.

**Comment by Rishit-dagli** [comment]
Do we want to try and merge this PR before the code freeze which starts on Monday?

**Review by sftim — COMMENTED** [review]
Thanks
/lgtm
/approve

**Pull request merged**